client/handler.go

```diff
@@ -307,6 +441,64 @@ func (h *Handler) Get(w http.ResponseWriter, r *http.Request, ps httprouter.Para
 	h.r.Writer().Write(w, r, c)
 }
 
+// swagger:route GET /dyn-clients/{id} public getDynOAuth2Client
```

Comment:
Make the client ID part of the query string as explained shown here: https://openid.net/specs/openid-connect-registration-1_0.html#ReadRequest

```suggestion
// swagger:route GET /connect/register public getOAuth2ClientPublic
```